manual_testcase/cart/cart.yaml

```diff
@@ -0,0 +1,34 @@
+Feature: Cart
+
+  Scenario: Apply Coupon when the coupon is valid
+    Given the user is on the Cart page
+    And the Cart has items
+    When the user enters a valid coupon in the Copoun field
+    And clicks the "Apply Coupon" button 
+    Then the sucess message "Success: Your coupon discount has been applied!" should appear
+    And the discount field should appear above the "Total" field 
```

Comment:
Suggestion: consider clarifying what the discount field displays (e.g., the value of the discount applied).